introduction.md

```diff
@@ -0,0 +1 @@
+This file introduces ti8m-hic to the deadly vipers dojo.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.